modules/docker-mirror/variables.tf

```diff
@@ -21,13 +21,13 @@ variable "machine_image" {
 
 variable "machine_type" {
   type        = string
-  default     = "n1-standard-2" // 2 vCPU, 7.5GB
+  default     = "n2-standard-2" // 2 vCPU, 8GB
   description = "Docker registry mirror node machine type."
 }
 
 variable "boot_disk_size" {
   type        = number
-  default     = 32
+  default     = 10
```

Comment:
We don't need more than the 10.

Reply:
it recreates it, like it does every time a new module is released because the base image changes - this does not affect executors as docker gracefully falls back to pulling from upstream hub.